src/utils/debounce.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Returns a function that will not be triggered after right after a call, waiting ${wait} seconds
+ * to be able to invoke it again. If ${immediate}, it will call again in the leading edge, instead of trailing.
+ * It's a adapted version of {@link http://underscorejs.org/#debounce}
```

Comment:
There's a lot of discussions after what happened with left-pad, but for sure we can use the lodash module, I think for the complexity it worth it :)